polling_stations/apps/data_collection/management/commands/import_kensington_and_chelsea.py

```diff
@@ -24,6 +24,24 @@ def station_record_to_dict(self, record):
             rec["location"] = Point(-0.200082, 51.491363, srid=4326)
             return rec
 
+            # Updates From Council
+            # Chelsea Old Town Hall, 165-181 King's Road, LONDON
+            record = record._replace(polling_place_uprnproperty_urn="217046329")
+            record = record._replace(polling_place_easting="527281")
+            record = record._replace(polling_place_northing="178093")
+
+        # Kensington Central Library, 12 Phillimore Walk, LONDON
+        if record.polling_place_id == "":
+            record = record._replace(polling_place_uprnproperty_urn="217068403")
+            record = record._replace(polling_place_easting="525421")
+            record = record._replace(polling_place_northing="179598")
+
+        # Christ Church, Victoria Road, LONDON, W8 5RQ
+        if record.polling_place_id == "":
+            record = record._replace(polling_place_uprnproperty_urn="217089088")
+            record = record._replace(polling_place_easting="525968")
+            record = record._replace(polling_place_northing="179187")
```

Comment:
this isn't going to cause a problem as such, but if we've got UPRN and E/N we're only going to use one of them. I'm guessing in these cases, it doesn't make much difference (I've not checked), but we might as well just set the one we want to use.